samples/tfm/tfm_cert/prj.conf

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BUILD_WITH_TFM=y
+CONFIG_TFM_PROFILE_TYPE_NOT_SET=y
+
+CONFIG_TFM_LOG_LEVEL_SILENCE=n
+CONFIG_TFM_SPM_LOG_LEVEL_DEBUG=y
+CONFIG_TFM_EXCEPTION_INFO_DUMP=y
+
+CONFIG_SECURE_BOOT=y
+CONFIG_BUILD_S1_VARIANT=n
+
+CONFIG_FW_INFO=y
+CONFIG_FW_INFO_FIRMWARE_VERSION=1
+
+# python3 nrf/scripts/bootloader/keygen.py --private -o priv.pem
+CONFIG_SB_SIGNING_KEY_FILE="priv.pem"
+
+# Enable mcumgr.
+CONFIG_MCUMGR=y
+
+# Some command handlers require a large stack.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2304
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Ensure an MCUboot-compatible binary is generated.
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+# Enable flash operations.
+CONFIG_FLASH=y
+
+# Required by the `taskstat` command.
+CONFIG_THREAD_MONITOR=y
+
+# Support for taskstat command
+CONFIG_OS_MGMT_TASKSTAT=y
+
+# Enable statistics and statistic names.
+CONFIG_STATS=y
+CONFIG_STATS_NAMES=y
+
+# Enable most core commands.
+CONFIG_MCUMGR_CMD_IMG_MGMT=y
+CONFIG_MCUMGR_CMD_OS_MGMT=y
+CONFIG_MCUMGR_CMD_STAT_MGMT=y
+
+# Enable logging
+CONFIG_LOG=y
+CONFIG_MCUBOOT_UTIL_LOG_LEVEL_WRN=y
+
+# Enable the serial mcumgr transport.
+CONFIG_MCUMGR_SMP_UART=y
```

Comment:
NB: When we are ready, we will need

CONFIG_TFM_NRF_PROVISIONING=y

It's disabled by default